waku/node/peer_manager/topic_health.nim

```diff
@@ -0,0 +1,19 @@
+import chronos
```

Comment:
I believe the `topic_health.nim` module should live under the `waku_relay` folder